docs/run.md

```diff
@@ -138,17 +138,19 @@ There are many use cases for the recorder.
 For instance:
 
 -   After having coded and tested contracts in a constrained
-    environment (like the JavaScript VM), you could then change the environment and redeploy it to a more realistic environment like a test net with an **injected web3** or to a Geth node. By using the generated **scenario.json** file, you will be using all the same settings that you used in the Javascript VM.  And this mean that you won't need to click the interface 100 times or whatever to get the state that you achieved originally.  So the recorder could be a tool to protect your sanity.
+    environment (like the Remix VM), you could then change the environment and redeploy it to a more realistic environment like a public testnet or to a Geth node. By using the generated **scenario.json** file, you will be using all the same settings that you used in the Remix VM.  And this means that you won't need to click the interface 100 times or whatever to get the state that you achieved originally.  So the recorder could be a tool to protect your sanity.
     
     You can also change the settings in the scenario.json file to customize the playback.
 
 -   Deploying contract does often require more than creating one
     transaction and so the recorder will automate this deployment.
 
--   Working in a dev environment often requires to setup the
+-   Working in a dev environment often requires setting up the
     state in a first place.  
 
-![](images/a-runtab-recorder.png)
+![](images/a-recorder.png)
+
+When checked, the option `Run transactions using the last compilation result` allows you to develop a contract and easily set the state.    
```

Comment:
This can be elaborated a bit more. When checked, recorder with run transaction with latest compiled copy of the contracts